packages/runtime/src/runtime.ts

```diff
@@ -5,14 +5,17 @@ import algosdk, {
 	decodeSignedTransaction,
 	modelsv2,
 	SignedTransaction,
+	Transaction,
 } from "algosdk";
 import cloneDeep from "lodash.clonedeep";
+import nacl from "tweetnacl";
```

Comment:
what does this lib does?

Reply:
Its a cryptographic library based on NaCl. Here are more details http://nacl.cr.yp.to